src/lib/get-link.ts

```diff
@@ -0,0 +1,63 @@
+import { SupportedLocale } from 'common/lib/locale';
+
+// English is always required, and the other locales are optional.
+type TranslatedLink = Partial< Record< SupportedLocale, string > > & { en: string };
+
+const DOCS_LINKS = {
+	studio: {
+		en: 'https://developer.wordpress.com/docs/developer-tools/studio/',
+		es: 'https://developer.wordpress.com/es/docs/herramientas-para-desarrolladores/studio/',
+	},
+	importExport: {
+		en: 'https://developer.wordpress.com/docs/developer-tools/studio/import-export/',
+		es: 'https://developer.wordpress.com/es/docs/herramientas-para-desarrolladores/studio/importar-exportar/',
+	},
+	sites: {
+		en: 'https://developer.wordpress.com/docs/developer-tools/studio/sites/',
+		es: 'https://developer.wordpress.com/es/docs/herramientas-para-desarrolladores/studio/sitios/',
+	},
+	sync: {
+		en: 'https://developer.wordpress.com/docs/developer-tools/studio/sync/',
+		es: 'https://developer.wordpress.com/es/docs/herramientas-para-desarrolladores/studio/sync/',
+	},
+	cli: {
+		en: 'https://developer.wordpress.com/docs/developer-tools/studio/cli/',
+	},
+} satisfies Record< string, TranslatedLink >;
+
+const BLOG_LINKS = {
+	blogPhpVersions: {
+		en: 'https://wordpress.com/blog/2025/03/17/studio-wordpress-php-versions/',
+		es: 'https://wordpress.com/es/blog/2025/04/02/modifica-las-versiones-de-wordpress-y-php-de-tu-sitio-local-con-studio/',
+		fr: 'https://wordpress.com/fr/blog/2025/03/28/studio-wordpress-php-versions/',
+		'pt-br': 'https://wordpress.com/pt-br/blog/2025/04/02/estudio-wordpress-php-versoes/',
```

Comment:
Having the full links makes it easier to test whether they work or return a 404.